test/e2e/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/onsi/ginkgo/v2 v2.9.2
 	github.com/onsi/gomega v1.27.6
 	github.com/pkg/errors v0.9.1
-	github.com/vmware-tanzu/tanzu-cli v0.0.0-00010101000000-000000000000
+	github.com/vmware-tanzu/tanzu-cli v0.90.0
```

Comment:
I feel it is better to keep using dummy `v0.0.0-00010101000000-000000000000` as we are not really using `v0.90.0` but the latest copy on the same branch to run the e2e tests because of the `replace` statement mentioned above. Updating to `v0.90.0` might give wrong impression to the user that we are using `v0.90.0` version here.


```
replace github.com/vmware-tanzu/tanzu-cli => ../..
```

Reply:
this no longer required